src/main/resources/css/reporting.css

```diff
@@ -49,6 +41,14 @@ a:hover {
     padding-left: 3px;
 }
 
+/* background elements will be moved closer to their respective scenarios.
+so don't want to indent again. keeps background and scenario visually coupled */
+.element.background {
+    margin-top: 0;
+    padding-left: 0;
+    box-shadow: none !important;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/2496596343/issues/source?bid=3059308&fileBranchId=3269761#l49)